app/src/main/java/com/example/memeapp/data/local/LocalMemeDatabaseImpl.kt

```diff
@@ -22,4 +25,18 @@ class LocalMemeDataBaseImpl() : LocalMemeDataBase {
         val memesRealm = realm.where(MemeRealmEntity::class.java).findAll()
         return memesRealm.transformRealmListToEntityList()
     }
+
+    override fun getSingleMeme(id: Int): MemesEntity {
+        val realm = Realm.getDefaultInstance()
+        val memesRealm = realm.where(MemeRealmEntity::class.java).findAll()
+        return (memesRealm.find { it.id == id })?.transformRealmToEntity() ?: MemesEntity(
```

Comment:
instead of initializing this like this. you should use default parameters

Reply:
Thx for the advice